templates/solidity-frameworks/foundry/packages/foundry/package.json

```diff
@@ -4,7 +4,7 @@
   "type": "module",
   "scripts": {
     "account": "make account",
-    "account:generate": "make account-generate ACCOUNT_NAME=${1:-scaffold-eth-custom}",
```

Comment:
this didn't work for me too but `make account-generate -- ACCOUNT_NAME=${1:-scaffold-eth-custom}` with `--` works as expected

Reply:
I just pushed a change cna you do `yarn generate` ? 

Make sure you do `yarn build:dev` on CLI

Reply:
thought about creating the issue but now it's fixed or you can try `--` version

Other parametrized scripts (fork, verify) could have the same problem. Didn't check them for now but at the first sight defatul params for both are wrong. We can tinker that later

Reply:
> I just pushed a change cna you do `yarn generate` ?
> 
> Make sure you do `yarn build:dev` on CLI

yes, it works as expected, thanks!

Reply:
<img width="1177" alt="image" src="https://github.com/user-attachments/assets/42584289-310c-4621-afa3-9670e262fa4f" />


Verify seems to work nicely, merging this for now 🙌